src/artikler/sok-sosialhjelp/SokSosialhjelpBokmal.tsx

```diff
@@ -112,27 +119,52 @@ const SokSosialhjelpBokmal: React.FC = () => {
                 ) && (
                     <>
                         <UnmountClosed isOpened={lesMer}>
-                            <div className="kommunesok_midlertidig">
-                                <KommuneSok
-                                    ledetekst="Sjekk om du kan søke digitalt i din kommune"
-                                    soknadTilgjengeligTekst="Du kan søke digitalt i"
-                                    soknadIkkeTilgjengelig={
-                                        <span>
-                                            kan dessverre ikke ta i mot digitale
-                                            søknader ennå. Du kan{" "}
-                                            <Lenke href={"./sok-papir?lang=nb"}>
-                                                søke på papirskjema
-                                            </Lenke>
-                                            .
-                                        </span>
-                                    }
-                                    placeholderTekst="Skriv kommunenavn"
-                                    ariaLabel="Søk etter kommunenavn"
-                                    onValgtKommune={(
-                                        kommuneId: string | undefined
-                                    ) => setKommuneId(kommuneId)}
-                                />
-                            </div>
+                            <KommuneSok
+                                ledetekst="Sjekk om du kan søke digitalt i din kommune"
+                                soknadOgInnsynTilgjengeligTekst={
+                                    <>
+                                        Du kan søke digitalt i{" "}
+                                        {valgtKommuneNavn}. Du kan også følge{" "}
+                                        <Lenke href="https://www.nav.no/sosialhjelp/innsyn">
+                                            statusen på søknaden
+                                        </Lenke>{" "}
+                                        din på nav.no
+                                    </>
+                                }
+                                soknadTilgjengeligUtenInnsynTekst={
+                                    <>
+                                        Du kan søke digitalt i{" "}
+                                        {valgtKommuneNavn}. Snart kan du også
```

Comment:
Litt usikker på denne teksten. Hva betyr det å snart kunne følge søknaden?

Reply:
"Snart kan du også følge status på søknaden din på nav.no" 
Synes det høres greit ut jeg?

Reply:
Det linkes jo til egen artikkelside som forklarer dette litt bedre. Jeg stemmer for at vi lar teksten være slik den er nå, og heller følger opp dersom feedback fra brukere viser at dette er uklart. :)